rectools/models/sasrec.py

```diff
@@ -21,6 +21,8 @@
 from rectools.models.rank import Distance, ImplicitRanker
 from rectools.types import InternalIdsArray
 
+LossDescription = tp.Literal["softmax", "BCE", "gBCE"]
```

Comment:
It's not a description at all  
Maybe `LossName`?

Reply:
fixed